index.html

```diff
@@ -0,0 +1,162 @@
+            <p>Inserted into Einstein’s equations, the latent energy would act as a kind of antigravity, and the universe would blow itself up. Since it was space itself supplying the repulsive force, the more space was created, the harder it pushed apart.</p>
+            <p>What would become our observable universe mushroomed in size at least a trillion trillionfold — from a submicroscopic speck of primordial energy to the size of a grapefruit — in less than a cosmic eye-blink.</p>
+            <p>Almost as quickly, this pulse would subside, relaxing into ordinary particles and radiation. All of normal cosmic history was still ahead, resulting in today’s observable universe, a patch of sky and stars billions of light-years across. “It’s often said that there is no such thing as a free lunch,” Dr. Guth likes to say, “but the universe might be the ultimate free lunch.”</p>
+            <p>Make that free lunches. Most of the hundred or so models resulting from Dr. Guth’s original vision suggest that inflation, once started, is eternal. Even as our own universe settled down to a comfortable homey expansion, the rest of the cosmos will continue blowing up, spinning off other bubbles endlessly, a concept known as the multiverse.</p>
+            <p>So the future of the cosmos is perhaps bright and fecund, but do not bother asking about going any deeper into the past.</p>
+            <p>We might never know what happened before inflation, at the very beginning, because inflation erases everything that came before it. All the chaos and randomness of the primordial moment are swept away, forever out of our view.</p>
+            <p>“If you trace your cosmic roots,” said Abraham Loeb, a Harvard-Smithsonian astronomer who was not part of the team, “you wind up at inflation.”</p>
+        </div>
+        <aside class="assidepos">
+            <h3>Editors Pick</h3>
+            <div class="aside-section">
+                <img class="aside-img" src="https://raw.githubusercontent.com/mariordgez/Project_1_HTML/project-1-microverse/images/pic9.PNG" alt="img9">
+                <p class="aside-text">If Everything Smells Bad, You are not alone</p>
+            </div>
+            <div class="aside-section">
+                <img class="aside-img" src="https://raw.githubusercontent.com/mariordgez/Project_1_HTML/project-1-microverse/images/pic10.PNG" alt="img10">
+                <p class="aside-text">What Does Home Mean to Us? Not the Same Thing It Did Before the Pandemic</p>
+            </div> 
+            <div class="aside-section">
+                <img class="aside-img" src="https://raw.githubusercontent.com/mariordgez/Project_1_HTML/project-1-microverse/images/pic11.PNG" alt="img11">
+                <p class="aside-text">Glimpses of Sudan’s Forgotten Pyramids</p>
+            </div> 
+        </aside>
+    </div>
```

Comment:
### Implement missing sections 
The following sections have not been implemented on the clone page. I have highlighted them below for easier reference. 

#### Suggested newsletters section

**NOTE: Do you mind also include in the implementation, the `READ 615 Comments` and the preceding social links as well?**

![Suggested-newsletters-section](https://user-images.githubusercontent.com/22345961/112322162-b77e0c00-8cc1-11eb-9d75-5167dba00c0f.png)

#### More into space and astronomy

![more-inspace-and-astronomy](https://user-images.githubusercontent.com/22345961/112322206-c4026480-8cc1-11eb-8411-08b31cad9b74.png)

#### Editors pick

![editors-pick-footer](https://user-images.githubusercontent.com/22345961/112322262-ce246300-8cc1-11eb-800f-4f53fb028cd6.png)

#### Footer
![footer](https://user-images.githubusercontent.com/22345961/112551750-8b05e500-8dd2-11eb-8f10-6606c81eaf6e.png)